### PR TITLE
Bootstrap Style Editing

### PR DESCRIPTION
**Pull request by phronmophobic**

### CODE REVIEW

File: pages/api/bootstrap_variables.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                                   APIView):
+    
+    def get(self, request):
+        serializer = BootstrapVariableSerializer(self.get_queryset(), many=True)
+        
+        return Response(serializer.data)
+                        
+    def put(self, request):
+        
+        print request.data
```
Comment (smirolo) [inline pages/api/bootstrap_variables.py:68]: Extra debugging message

File: pages/api/bootstrap_variables.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+LOGGER = logging.getLogger(__name__)
+
+
+
+class BootstrapVariableMixin(AccountMixin):
+
+    pass
+
+
+class BootstrapVariableListAPIView(AccountMixin,
```
Comment (smirolo) [inline pages/api/bootstrap_variables.py:58]: Why not derive from restframework.generics.ListCreateAPIView?

**Comment by smirolo** [comment]
Good to merge. Thank you!

File: pages/api/bootstrap_variables.py (hunk 1 above)
Comment (phronmophobic) [inline pages/api/bootstrap_variables.py:68]: oops.

File: pages/api/bootstrap_variables.py (hunk 2 above)
Comment (phronmophobic) [inline pages/api/bootstrap_variables.py:58]: I tried that, but couldn't get it to work right. The POST would add a new row, but what I wanted to do was replace the whole list with a new list. There's probably a better way to do it, but I couldn't figure out how to get the ListCreateAPI to work on multiple elements.

**Pull request merged**